packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -42,29 +43,32 @@ class DebuggerController extends DisposableController
     if (routerDelegate != null) {
       codeViewController.subscribeToRouterEvents(routerDelegate);
     }
-
-    evalService = EvalService(
-      isolateRef: isolateRef,
-      variables: variables,
-      frameForEval: () =>
-          _selectedStackFrame.value?.frame ??
-          _stackFramesWithLocation.value.safeFirst?.frame,
-      isPaused: isPaused,
+    addAutoDisposeListener(_selectedStackFrame, () => _updateCurrentFrame());
+    addAutoDisposeListener(
+      _stackFramesWithLocation,
+      () => _updateCurrentFrame(),
```

Comment:
nit: why is this
`() => _updateCurrentFrame()`
instead of
`_updateCurrentFrame`
?